### ISSUE TITLE
Couldn't process kernel message TypeError: Cannot read property 'output_type' of undefined

### ISSUE THREAD

**Issue by shingo78** [NII-cloud-operation/Jupyter-multi_outputs#14]
TypeError occurs in _handle_execute_reply, if you execute as follows.

![image](https://user-images.githubusercontent.com/10056150/29812100-84abd3aa-8ce0-11e7-8efc-4b25ac5de9c8.png)

```
kernel.js:1007 Couldn't process kernel message TypeError: Cannot read property 'output_type' of undefined
    at CodeCell.codecell.CodeCell._handle_execute_reply (main.js?v=20170829111719:247)
    at i (jquery.min.js:4)
    at callbacks.shell.reply (main.js?v=20170829111719:77)
    at Object.callbacks.shell.reply (main.js?v=20170829111719:182)
    at kernel.js:1040
    at <anonymous>
(anonymous) @ kernel.js:1007
Promise rejected (async)
Kernel._handle_ws_message @ kernel.js:1007
i @ jquery.min.js:4
```
https://github.com/NII-cloud-operation/Jupyter-multi_outputs/blob/master/main.js#L247

**Comment by shingo78** [comment]
This error occurs randomly.
I assume that it depends on the order of messages from the kernel.

**Comment by yacchin1205** [comment]
Fix by #17
